LadderManager.cpp

```diff
@@ -279,9 +315,9 @@ void StartBotProcess(std::string CommandLine)
 			exitCode = -1;
 		}
 
-
 		// We succeeded.
 	}
+	CloseHandle(h);
```

Comment:
It might be good to check for the result of this call and log it to stderr. It could help spot an issue in the future.
Also: will this invalidate the file handle for the created process? Might want to check this if you haven't already.

Reply:
It will close the handle. But the created process is already over here.

Reply:
Oh sorry, didn't see the call to `WaitForSingleObject()`
No problem then.